### PR TITLE
Create Desktop version using media queries on mobile version of project

### PR DESCRIPTION
**Pull request by fibini**
Used media queries on mobile version to create the Desktop version, changed and added code to index.html and css files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -155,10 +177,11 @@ <h1 class="post">Multi-Post Stories</h1>
...
           <li>Ruby</li>
         </ul>
-        <div class="but">
+        <div class="but2">
           <button class="bt" type="button">see project</button>
         </div>
       </section>
+      <!--Middle3-->
       <section class="middle3">
         <img
```
Comment (youmari) [inline index.html:186]: Nice implementing the design, however, you should add some padding with an 804px widescreen so the left side will be the same as the right side 
image

File: index.html [hunk 2]
```diff
@@ -29,45 +36,54 @@ <h2 class="soft">Software Developer</h2>
...
+              </ul>
+            </div>
           </div>
         </div>
-        <img src="./img/Devider.png" alt="" />
+        <img class="devii" src="./img/Devider.png" alt="" />
+        <div class="devi">
+          <img class="vector-4" src="./img/Vector4.png" alt="thin grey line" />
+        </div>
         <div class="socials">
```
Comment (youmari) [inline index.html:82]: please try to remove the yellow color when hovering on the links 
image

**Review by youmari — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights

✔️ Great Design implementation
✔️No linter error
✔️ Descriptive PR


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fibini** [comment]
Made recommended changes to project, added padding to middle section and removed yellow hover effect from social icons

**Review by OLIPLICHE — APPROVED** [review]
Hi @fibini , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**